### DISCUSSION TITLE
Is there a way exclude regional-only holidays from outside of my region?

### DISCUSSION THREAD

**Discussion by rafalkrupinski** [GothenburgBitFactory/timewarrior#519] [Q&A] [answered]
All national and regional holidays are excluded in the calendar view. It also affects the available time.

Is there a way to only have national + my regions holidays?

**Comment by lauft** [comment, marked as answer]
What do you mean by 'calendar view'? (Taskwarrior has a [calendar report](https://taskwarrior.org/docs/commands/#fixed-reports)... 🤔)

Timewarrior has the `summary` report and the charts `day`, `week`, and `month` which show holidays as exclusions. Those holidays are defined either in your config file or a dedicated holiday file which is imported there. So if you do not want a holiday to show up, you just delete it there.
